src/irc/__init__.py

```diff
@@ -57,6 +59,7 @@ def cli(
     async def connect(**kwargs):
         bot.send("NICK", nick=irc_nick)
         bot.send("USER", user=irc_nick, realname=irc_realname)
+        bot.send("PASS", password=irc_password)
```

Comment:
Thanks! This needed.  Could you make it conditional?

```suggestion
if irc_password is not None:
        bot.send("PASS", password=irc_password)
```